### PR TITLE
authorino version v0.18.0

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -72,6 +72,7 @@ require (
 	github.com/exponent-io/jsonpath v0.0.0-20210407135951-1de76d718b3f // indirect
 	github.com/fatih/color v1.17.0 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
+	github.com/foxcpp/go-mockdns v1.1.0 // indirect
```
Comment (guicassolato) [inline go.mod:75]: It feels like this is not strictly related to bumping `github.com/kuadrant/authorino`, but I guess it's OK?

File: go.mod [hunk 2]
```diff
@@ -135,15 +136,13 @@ require (
...
+	github.com/sergi/go-diff v1.3.1 // indirect
 	github.com/shopspring/decimal v1.4.0 // indirect
 	github.com/sirupsen/logrus v1.9.3 // indirect
 	github.com/spf13/cast v1.6.0 // indirect
 	github.com/spf13/cobra v1.8.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/telepresenceio/watchable v0.0.0-20220726211108-9bb86f92afa7 // indirect
-	github.com/tidwall/gjson v1.14.0 // indirect
-	github.com/tidwall/match v1.1.1 // indirect
-	github.com/tidwall/pretty v1.2.0 // indirect
```
Comment (guicassolato) [inline go.mod:146]: These ones disappearing from the list of dependencies kinds of surprises me. Has Go become smarter on knowing this part of Authorino's code is not reached by the Kuadrant Operator maybe?

Comment (eguzki) [reply go.mod:146]: blame `go mod tidy`

Comment (guicassolato) [reply go.mod:146]: OK. I'm gonna trust this is good news.

**Review by guicassolato — APPROVED** [review]

**Comment by eguzki** [comment]
Only failing license compliance analysis. This is due to Fossa issue.. I have re run the analysis 3 times, however, it seems to be stuck. 

Screenshot 2024-10-02 at 14-43-46 kuadrant-operator - Projects FOSSA

Merging this regardless of the license compliance analysis
